NetworkConfigurations/TrafficControlAndPolicingSubCategories/PortMirroring.md

```diff
@@ -0,0 +1,79 @@
+---
+title: Port Mirroring
+grand_parent: Network Configuration
+parent: Traffic Control and Policing
+nav_order: 5
+layout: default
+---
+
+# Port Mirroring
+
+Port mirroring is a powerful feature that allows you to copy packets traversing a physical switch port and send them to another port for analysis. This is crucial for monitoring and troubleshooting network traffic without disrupting normal operations.
+
+### Configuring Port Mirroring
```

Comment:
How mirroring can be configured is well documented. However, I am not convinced it has been tested. Please add a section showing how you've verified this configuration. Sample output works the best.

Reply:
I'll add the outputs.